### PR TITLE
[ADD] Recruitment: adding new recruitment flow doc

### PR DESCRIPTION
**Pull request by larm-odoo**
Original Recruitment doc too long, breaking off the recruitment flow section into its own doc.

[Task Card](https://www.odoo.com/odoo/project/3835/tasks/3967973?cids=3) for this PR.

### CODE REVIEW

**Comment by larm-odoo** [comment]
Hi @Felicious - this is ready for a first-round review!

**Review by Felicious — COMMENTED** [review]
Wow, @larm-odoo, I'm really blown away by the thoroughness of this document covering the entire recruitment workflow! Your effort really shines through. 😊 

I'm not quite done with my review yet, but I wanted to share some initial thoughts before our meeting tomorrow. Looking forward to collaborating on breaking this use case into several more general pieces of the recruitment workflow!

**Comment by larm-odoo** [comment]
Hi @Felicious - now that all the other recruitment docs have been added, this is ready for another look! There were a lot of build errors I had to fix to catch this up to the latest version. I think it's in good shape, but as always- your feedback is amazing- so if you have a chance, I'd love another review.

File: content/applications/hr/recruitment/recruitment-flow.rst
Comment (Felicious) [inline content/applications/hr/recruitment/recruitment-flow.rst:22]: maybe add a link to the "add new applicants" doc here?

File: content/applications/hr/recruitment/recruitment-flow.rst
Comment (Felicious) [inline content/applications/hr/recruitment/recruitment-flow.rst:39]:
```suggestion
If a resumé was attached to the online application, it appears in the attachments
```

File: content/applications/hr/recruitment/recruitment-flow.rst
Comment (Felicious) [inline content/applications/hr/recruitment/recruitment-flow.rst:41]: totally optional suggestion for brevity! (:
```suggestion
documents, navigate to the main :menuselection:`Documents`
```

File: content/applications/hr/recruitment/recruitment-flow.rst
Comment (Felicious) [inline content/applications/hr/recruitment/recruitment-flow.rst:43]: Totally optional!
```suggestion
dashboard, and click the :guilabel:`Recruitment` folder on the left-hand side.
```

File: content/applications/hr/recruitment/recruitment-flow.rst
Comment (Felicious) [inline content/applications/hr/recruitment/recruitment-flow.rst:55]:
```suggestion
The *Surveys* application is required to send interviews to applicants, so it must be installed.
```

File: content/applications/hr/recruitment/recruitment-flow.rst
Comment (Felicious) [inline content/applications/hr/recruitment/recruitment-flow.rst:70]:
```suggestion
To send an interview to an applicant, click an applicant's card to open detailed information about
the applicant. At the top-left of the applicant's card, click the :guilabel:`Send Interview`
button.
```

File: content/applications/hr/recruitment/recruitment-flow.rst
Comment (Felicious) [inline content/applications/hr/recruitment/recruitment-flow.rst:92]: In this section, I like how you've already referenced other docs to cut down on the length :smile: 

This is entirely optional and a suggestion to further cut down on length:

Is this information and the specific instructions for these actions covered in the Schedule interviews doc? If so, to avoid repeating information, maybe we can just summarize it. 

For example, for instructions like "click here to do this", we could rewrite it as "when sending emails, additional recipients can be added" and skip the step-by-step details.

**Review by Felicious — APPROVED** [review]
AMAZING JOB @larm-odoo! 

This doc has come a long way after my initial suggestion spawned so many additional PRs. I only had a few very minor suggestions, and I wanted to show my appreciation one more time for taking my suggestions so graciously and seriously to break up the PR into several different docs. The info is much easier to find now, and your attention to detail is commendable!

File: content/applications/hr/recruitment/recruitment-flow.rst
Comment (larm-odoo) [reply content/applications/hr/recruitment/recruitment-flow.rst:92]: I actually double checked, and the information is not the same. I also checked to see if there as a general 'this is ow you send emails' doc, but there isn't one of those either. The schedule interviews one is all about starting in the calendar, and all the various specifications for that- so a lot of the fields don't match/is different enough that I think this is OK to keep in here. I completely understand (and agree) with your suggestion to cut down on the 'overly detailed' information, but I honestly don't see it anywhere else that I can link it to, so I think I will keep it for now.

**Comment by larm-odoo** [comment]
Hi @ksc-odoo - this is ready for a final review!

File: content/applications/hr/recruitment/recruitment-flow.rst
Comment (ksc-odoo) [inline content/applications/hr/recruitment/recruitment-flow.rst:175]:
```suggestion
applicant's card to the :guilabel:`Second Interview` stage from the Kanban view of the :guilabel:`Applicants` page, or click on the :guilabel:`Second
```

File: content/applications/hr/recruitment/recruitment-flow.rst
Comment (ksc-odoo) [inline content/applications/hr/recruitment/recruitment-flow.rst:179]:
```suggestion
activities or emails configured for this stage, by default. The recruiter can now :ref:`schedule a second
```

File: content/applications/hr/recruitment/recruitment-flow.rst
Comment (ksc-odoo) [inline content/applications/hr/recruitment/recruitment-flow.rst:188]: Do you think it would be beneficial to mention moving it to the Contract Proposal stage (in the same way you've described moving it to other stages, up to this point)? Also, is there any benefit to discussing the folded 'Contract Signed' stage? I think if those were added, it could fully round out the entire flow you are describing. What do you think?

**Review by ksc-odoo — CHANGES_REQUESTED** [review]
hey there, @larm-odoo -- just finished my Final Review of this PR. Nice job. There are a decent amount of suggestions/feedback/comments/etc. that require your attention. And I think if you address those, and implement the necessary changes, you can tag me again for another quick look. Thanks! 👍

File: content/applications/hr/recruitment/recruitment-flow.rst
Comment (larm-odoo) [reply content/applications/hr/recruitment/recruitment-flow.rst:65]: I included both to show that you can make an "interview" (which is an odd sounding term to me=) have both technical testing and also innocuous non-"testing" types of questions like scheduling.

File: content/applications/hr/recruitment/recruitment-flow.rst
Comment (larm-odoo) [reply content/applications/hr/recruitment/recruitment-flow.rst:121]: True- I always tend to add too much!

File: content/applications/hr/recruitment/recruitment-flow.rst
Comment (larm-odoo) [reply content/applications/hr/recruitment/recruitment-flow.rst:137]: I believe I made the changes you mention above- but if I didn't fully understand, just let me know (the Kanban drag and drop is still mentioned first- so I wasn't sure if you meant for me to delete that as well?).

File: content/applications/hr/recruitment/recruitment-flow.rst
Comment (larm-odoo) [reply content/applications/hr/recruitment/recruitment-flow.rst:188]: I was thinking about that but I do have that step included in that document, and always worry about the doubling of information. I will add that though, just so this is thorough!

File: content/applications/hr/recruitment/recruitment-flow.rst
Comment (larm-odoo) [reply content/applications/hr/recruitment/recruitment-flow.rst:42]: I double checked right now, and I checked in the database I made for my recruitment tour. In a blank database, it does go to the Recruitment folder. I honestly do not know why it does not do that in Runbot, unless they made some modifications.

This is why my docs take forever, because  always have to compare to a blank/fresh DB and the runbots, since there are so many configs in Runbot =D! But thank you as always for double checking how everything works- I really appreciate it!

File: content/applications/hr/recruitment/recruitment-flow.rst
Comment (larm-odoo) [reply content/applications/hr/recruitment/recruitment-flow.rst:25]: OMG you are right! You have to add EITHER a LinkedIN OR a Resume!!! I think this must have been changed because I swear, every time I was testing in my own blank DB's for my tour I always had to enter a fake LinkedIn profile. This is why another set of eyes is always helpful!

**Comment by larm-odoo** [comment]
Hi @ksc-odoo - thank you for such a thorough and thoughtful review! I believe I got everything you suggested edited, and answered any questions you had. This is ready for another look!

File: content/applications/hr/recruitment/recruitment-flow.rst
Comment (ksc-odoo) [inline content/applications/hr/recruitment/recruitment-flow.rst:43]:
```suggestion
To find the recruitment documents, navigate to the main :menuselection:`Documents app` dashboard, and
```

File: content/applications/hr/recruitment/recruitment-flow.rst
Comment (ksc-odoo) [inline content/applications/hr/recruitment/recruitment-flow.rst:48]:
```suggestion
<recruitment/settings>` of the *Recruitment* app, the resumé appears on the applicant's card,
```

File: content/applications/hr/recruitment/recruitment-flow.rst
Comment (ksc-odoo) [inline content/applications/hr/recruitment/recruitment-flow.rst:53]: ...and if it appears below, it appears as a PDF link/button -- it doesn't just readily present itself

File: content/applications/hr/recruitment/recruitment-flow.rst
Comment (ksc-odoo) [inline content/applications/hr/recruitment/recruitment-flow.rst:75]:
```suggestion
To send an interview to an applicant, first click the applicant's card from the :guilabel:`Applications`
```

File: content/applications/hr/recruitment/recruitment-flow.rst
Comment (ksc-odoo) [inline content/applications/hr/recruitment/recruitment-flow.rst:95]:
```suggestion
Sometimes, pre-configured email templates in Odoo use dynamic placeholders, which are automatically
```

File: content/applications/hr/recruitment/recruitment-flow.rst
Comment (ksc-odoo) [inline content/applications/hr/recruitment/recruitment-flow.rst:102]:
```suggestion
emails` field, if more people should receive the email. If an email address is in the database as a contact,
```

File: content/applications/hr/recruitment/recruitment-flow.rst
Comment (ksc-odoo) [inline content/applications/hr/recruitment/recruitment-flow.rst:104]:
```suggestion
not in the database as a contact, and they should **not** be added as a contact, add their email address in
```

File: content/applications/hr/recruitment/recruitment-flow.rst
Comment (ksc-odoo) [inline content/applications/hr/recruitment/recruitment-flow.rst:109]:
```suggestion
attach it to the email. The attachment loads, and is listed above the :icon:`fa-paperclip`
```

File: content/applications/hr/recruitment/recruitment-flow.rst
Comment (ksc-odoo) [inline content/applications/hr/recruitment/recruitment-flow.rst:118]:
```suggestion
click on the desired day to select the date.
```

File: content/applications/hr/recruitment/recruitment-flow.rst
Comment (ksc-odoo) [inline content/applications/hr/recruitment/recruitment-flow.rst:140]:
```suggestion
This stage exists to quickly sort candidates that have potential, from those that do not meet the
```

File: content/applications/hr/recruitment/recruitment-flow.rst
Comment (ksc-odoo) [inline content/applications/hr/recruitment/recruitment-flow.rst:142]:
```suggestion
informs the recruitment team to potentially set up a phone call (or an interview) with the candidate.
```

File: content/applications/hr/recruitment/recruitment-flow.rst
Comment (ksc-odoo) [inline content/applications/hr/recruitment/recruitment-flow.rst:146]:
```suggestion
   be dragged and dropped in the Kanban view of the :guilabel:`Applications` page to the desired
```

File: content/applications/hr/recruitment/recruitment-flow.rst
Comment (ksc-odoo) [inline content/applications/hr/recruitment/recruitment-flow.rst:174]:
```suggestion
the :guilabel:`First Interview` stage on the status bar at the top of the individual applicant's card.
```

File: content/applications/hr/recruitment/recruitment-flow.rst
Comment (ksc-odoo) [inline content/applications/hr/recruitment/recruitment-flow.rst:198]:
```suggestion
:guilabel:`Applications` page, or click on the :guilabel:`Second Interview` stage at the top of
```

File: content/applications/hr/recruitment/recruitment-flow.rst
Comment (ksc-odoo) [inline content/applications/hr/recruitment/recruitment-flow.rst:215]:
```suggestion
Proposal` stage from the Kanban view of the :guilabel:`Applications` page, or click on the
```

File: content/applications/hr/recruitment/recruitment-flow.rst
Comment (ksc-odoo) [inline content/applications/hr/recruitment/recruitment-flow.rst:216]:
```suggestion
:guilabel:`Contract Proposal` stage at the top of the individual applicant's card.
```

File: content/applications/hr/recruitment/recruitment-flow.rst
Comment (ksc-odoo) [inline content/applications/hr/recruitment/recruitment-flow.rst:224]:
```suggestion
Drag-and-drop the applicant's card to the :guilabel:`Contract Signed` stage from the Kanban view of
```

File: content/applications/hr/recruitment/recruitment-flow.rst
Comment (ksc-odoo) [inline content/applications/hr/recruitment/recruitment-flow.rst:225]:
```suggestion
the :guilabel:`Applications` page, or click the :icon:`fa-ellipsis-h` :guilabel:`(ellipsis)` icon at
```

File: content/applications/hr/recruitment/recruitment-flow.rst
Comment (ksc-odoo) [inline content/applications/hr/recruitment/recruitment-flow.rst:226]:
```suggestion
the top of the individual applicant's card, then click :guilabel:`Contract Signed` on the status bar.
```

**Review by ksc-odoo — APPROVED** [review]
hey @larm-odoo -- thanks for making those adjustments. I have another handful of comments/feedback/etc. that require your attention. But, once you make those final adjustments/modifications, you can feel free to tag this for Tech Review. Thanks! 👍

File: content/applications/hr/recruitment/recruitment-flow.rst
Comment (larm-odoo) [reply content/applications/hr/recruitment/recruitment-flow.rst:224]: Ah! I missed one!

**Comment by larm-odoo** [comment]
Hi @samueljlieber - this is ready for a tech review, thanks!

File: content/applications/hr/recruitment/recruitment-flow.rst
Comment (StraubCreative) [inline content/applications/hr/recruitment/recruitment-flow.rst:87]: nitpick 😛 
this gui for **Edit:(Applicant's Name)** does *not* have a space after the colon, however there *is* a space in the same gui on line 92

**Review by StraubCreative — APPROVED** [review]
LGTM, nice job @larm-odoo 
Only a nitpick here, can address if you agree.
For a follow-up PR on this doc, consider expanding out the latter stages and also including a section on Stage Automations that would be relevant for recruiters 💪 
.....
@robodoo delegate=larm-odoo

**Comment by larm-odoo** [comment]
@robodoo r+

**Pull request closed without merging**
